### PR TITLE
add new terms genetic muscle disorder and metabolic muscle disorder

### LINKED ISSUE #5658: [NTR/gene] Hereditary inclusion-body myopathy
**Preferred gene-related syndrome label**
[NTR/gene] Hereditary inclusion-body myopathy

See
https://pubmed.ncbi.nlm.nih.gov/25149037/

rename inclusion myopathy (allof the diseases are hereditary).
Also, 
revise congenital myopathy according to https://ijponline.biomedcentral.com/articles/10.1186/s13052-017-0419-z

Also add term 
Genetic Muscle Disorders:
with these three main children
[Muscular dystrophies](https://www.kennedykrieger.org/patient-care/conditions/muscular-dystrophy): Duchenne, Becker, Facioscapulohumeral, Myotonic, Congenital, Distal, Emery-Dreifuss, Oculopharyngeal, Limb Girdle
    [Congenital myopathies](https://www.kennedykrieger.org/patient-care/conditions/congenital-myopathy): Central Core, Myotubular, Nemaline, Ullrich/Bethlem, RyR1
    [Metabolic muscle diseases](https://www.kennedykrieger.org/patient-care/conditions/metabolic-myopathies): Mitochondrial Myopathy, Pompe Disease, McArdles Disease, Carnitine Palmitoyl Transferase Deficiency

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #5658

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
@sabrinatoro could you review this PR with @pnrobinson when you meet on Thursday? thank you!

**Comment by sabrinatoro** [comment]
Addresses https://github.com/monarch-initiative/mondo/issues/6008

**Review by sabrinatoro — APPROVED** [review]

**Comment by sabrinatoro** [comment]
@nicolevasilevsky I made a lot of changes to this PR based on our expert discussion on March 2, 2023. 
Please take a look and review before merging. 
I think we can merge and continue the work in another PR and on the other issue (https://github.com/monarch-initiative/mondo/issues/6008).

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -440074,8 +440092,9 @@ xref: MESH:C536694 {source="UMLS:C1279186", source="MONDO:equivalentTo"}
 xref: OMIM:143100 {source="GARD:0005557", source="MONDO:relatedTo"}
 xref: SCTID:182747006 {source="UMLS:C1279186", source="MONDO:equivalentTo"}
 xref: UMLS:C1279186 {source="GARD:0005557", source="MONDO:equivalentTo"}
-is_a: MONDO:0000995 ! familial periodic paralysis
+is_a: MONDO:0007739 ! Huntington disease
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:440095]: @sabrinatoro when there are new superclass assertions, could you add Peter's ORCID and also add the term tracker item to the ticket?

thank you!

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -463666,6 +463686,21 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
...
+id: MONDO:0700223
+name: hereditary skeletal muscle disorder
+def: "An instance of muscle tissue disorder that is caused by an inherited genomic modification in an individual." [MONDO:patterns/hereditary]
+synonym: "genetic muscle disease" EXACT [https://orcid.org/0000-0001-5208-3432]
+synonym: "genetic muscle disorder" EXACT []
+synonym: "genetic muscular disease" EXACT [https://orcid.org/0000-0001-5208-3432]
+synonym: "genetic muscular disorder" EXACT [https://orcid.org/0000-0001-5208-3432]
+synonym: "hereditary muscle disorder" EXACT [https://orcid.org/0000-0001-5208-3432]
+is_a: MONDO:0003847 {source="https://orcid.org/0000-0002-0736-9199"} ! hereditary disease
+is_a: MONDO:0020120 ! skeletal muscle disorder
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:463699]: can you please add Peter's ORCID here too?

**Review by nicolevasilevsky — COMMENTED** [review]
Could you please make the couple minor changes, then this can be merged. Thank you!

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:440095]: done

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:463699]: done

**Pull request merged**